readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page and hovers
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_hovers/)
+- [DEMO LINK](https://prostobadobobb.github.io/layout_catalog_hovers/)
```

Comment:
The demo link doesn't work. Please try to redeploy it. Ask in the chat if you have questions